robot/server/server.go

```diff
@@ -466,8 +466,10 @@ func (s *Server) GetCloudMetadata(ctx context.Context, _ *pb.GetCloudMetadataReq
 		return nil, err
 	}
 	return &pb.GetCloudMetadataResponse{
-		RobotPartId:  md.RobotPartID,
-		LocationId:   md.LocationID,
-		PrimaryOrgId: md.PrimaryOrgID,
+		RobotPartId:   md.MachinePartID, // Deprecated: duplicates MachinePartId
```

Comment:
not sure if we need a follow-up to remove

Reply:
ya can you make/link the ticket here?